bundles/org.openhab.ui/web/src/css/app.styl

```
@@ -14,6 +14,7 @@ html
   --f7-list-item-header-line-height 1.3
   --f7-label-line-height 1.3
   --f7-list-item-after-line-height 1.3
+  --f7-list-item-min-height 36px
```

Hmm thanks. Can you then try it with this code?

```suggestion
  .device-macos .chrome
    --f7-list-item-min-height 36px
```

---

This is what I have:

```
.device-desktop
  --f7-navbar-title-line-height 1.3
  --f7-navbar-subtitle-line-height 1.3
  --f7-subnavbar-title-line-height 1.3
  --f7-list-item-header-line-height 1.3
  --f7-label-line-height 1.3
  --f7-list-item-after-line-height 1.3
  .device-macos .chrome
    --f7-list-item-min-height 36px
```

But this didn't work.

---

Hmm in that case leave out `.chrome` and try again. 

@ghys Do you know how to target only specific browsers in CSS?

---

Removing .chrome doesn't work either:

```
.device-desktop
  --f7-navbar-title-line-height 1.3
  --f7-navbar-subtitle-line-height 1.3
  --f7-subnavbar-title-line-height 1.3
  --f7-list-item-header-line-height 1.3
  --f7-label-line-height 1.3
  --f7-list-item-after-line-height 1.3
  .device-macos
    --f7-list-item-min-height 36px
```

Just .chrome without .device-macos doesn't work either.